DeviceAdministration/Web/Scripts/Views/Dashboard/TelemetryHistory.js

```diff
@@ -11,6 +11,13 @@
         var targetControlTitle;
         var visual;
         var width;
+        var hasVisualBeenInitialized = false;
+        
+        var reservedDataColumns = [
+            "timestamp",
+            "deviceId",
+            "externalTemperature" // TODO: remove this
```

Comment:
Currently, we get back `externalTemperature` with the data returned from the API, but it's null.